### ISSUE TITLE
Read more link for section 6.11. Support blacklisting JWTs is Dead

### ISSUE THREAD

**Issue by manishsaraan** [i0natan/nodebestpractices#240]
I tried to open the **Read More: Blacklist JSON Web Tokens** link in security section but its throwing 404.

**Comment by js-kyle** [comment]
Hi @manishsaraan thanks for pointing this out! Will fix this asap

**Comment by js-kyle** [comment]
@manishsaraan Beat me to it! Thanks

**Comment by Trewaters** [comment]
This happened to me today. I tried to follow the link and got a 404. I don't the see the `revokejwt.md` file in the repo. 

I really like this repo. Many thanks to everyone working to maintain it.

**Comment by js-kyle** [comment]
@Trewaters Hi there, thanks for the comment!

I've just read your update - I can't see any references to the incorrect link in the master branch (the medium article also follows the correct link). I'm interested to hear if you work out how you came across the old link!

**Comment by Trewaters** [comment]
I checked the source code of the medium article ( https://medium.com/@nodepractices/were-under-attack-23-node-js-security-best-practices-e33c146cb87d ) and 4 spots refer to the draft. I unknowingly saved that one when I started. Now I have the updated link.

the link I followed was "https://github.com/i0natan/nodebestpractices/tree/security-best-practices-section/sections/security".

Thanks for getting back with me. I will be crediting this article in my talk tonight at SFNode ( https://www.meetup.com/sfnode/events/mxstsmyxpbcb/ ).
